src/ui/components/Carousel/index.js

```diff
@@ -0,0 +1,48 @@
+/* @flow */
+import React from 'react';
+import NukaCarousel from 'nuka-carousel';
+
+import Card from 'ui/components/Card';
+
+import './styles.scss';
+
+
+type PropTypes = {|
+  sections: Array<any>,
+|};
+
+const Carousel = ({ sections }: PropTypes) => {
+  if (!sections) {
+    throw new Error('sections are required for a Carousel component');
+  }
+
+  return (
+    <Card className="Carousel">
+      <NukaCarousel
+        autoplay={false}
+        autoplayInterval={4000}
+        cellAlign="left"
+        cellSpacing={10}
+        framePadding="0 10%"
+        frameOverflow="visible"
+        slidesToShow={1}
+        slidesToScroll={1}
+        slideWidth={1}
+      >
+        {(sections.map((section, i) => {
+          // We have to wrap this content in a <div> or ReactSlick
```

Comment:
ReactSlick is gone. Does `NukaCarousel` also need this?

Reply:
Yeah I think so, I'll double-check though.